target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -155,6 +155,16 @@ archer-c7-v4)
 	ucidef_set_led_usbdev "usb1" "USB1" "$board:green:usb1" "1-1"
 	ucidef_set_led_usbdev "usb2" "USB2" "$board:green:usb2" "2-1"
 	;;
+archer-c7-v5)
```

Comment:
since its almost identical to the archer-c7-v4 , can you merge them?

Reply:
but archer-c7-v5 doesn't have usb2.

Reply:
```
case "$board" in 
    archer-c7-v4)
       ucidef_set_led_usbdev "usb2" "USB2" "$board:green:usb2" "2-1"
```

Reply:
done

Reply:
Done